capstone/src/main/java/com/step902020/capstone/EventRepository.java

```diff
@@ -1,11 +1,14 @@
 package com.step902020.capstone;
+
 import java.util.List;
 import org.springframework.cloud.gcp.data.datastore.repository.DatastoreRepository;
 import org.springframework.core.convert.converter.Converter;
 
-/**
- * 
- */
-public interface EventRepository extends DatastoreRepository<EventTemp, Long> {
-    public EventTemp findById(long id);
-}
\ No newline at end of file
+public interface EventRepository extends DatastoreRepository<Event, Long> {
+
+  public List<Event> findByDatastoreID(Long datastoreID);
```

Comment:
I believe this is equivalent to `findById` which already exists in the `DatastoreRepository`.

Reply:
Ah! It likely is. I've misled @oasemota99 by changing the custom interface method from `findByID` to `findByDatastoreID`, but removing the method entirely and using the built-in one is likely a better solution!